clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/ReadGroupElasticsearchDAOSpec.scala

```diff
@@ -20,7 +20,7 @@ class ReadGroupElasticsearchDAOSpec
   }
 
   it should "updateReadGroupMetadata" in {
-    val key = ModelReadGroupKey("barcodeURGM1", 2, "library3")
+    val key = ModelReadGroupKey("barcodeURGM1", 2, "library3", "GCP")
```

Comment:
I think the location values should be enumerated, but I'm not sure how or where.